<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="iOSTest"
           tests="3"
           failures="2"
           errors="0"
           skipped="1"
           time="0.020">
  <properties />
            <testcase name="after_Suite" time="0.000" classname="ios.scripts.Test_04_UI_Catalog_AlertViews">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  ios.common.functions.Base_Class.after_Suite(Base_Class.java:79)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:357)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
  org.testng.TestNG.runSuites(TestNG.java:1133)
  org.testng.TestNG.run(TestNG.java:1104)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="test_04_UI_Catalog_AlertViews" time="0.003" classname="ios.scripts.Test_04_UI_Catalog_AlertViews">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  ios.pageObjects.Get_WebElement.getElementByAccessibilityId(Get_WebElement.java:22)
  ios.scripts.Test_04_UI_Catalog_AlertViews.test_04_UI_Catalog_AlertViews(Test_04_UI_Catalog_AlertViews.java:35)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
  org.testng.internal.Invoker.retryFailed(Invoker.java:998)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:756)
  org.testng.TestRunner.run(TestRunner.java:610)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
  org.testng.SuiteRunner.run(SuiteRunner.java:289)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
  org.testng.TestNG.runSuites(TestNG.java:1133)
  org.testng.TestNG.run(TestNG.java:1104)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
            <testcase name="test_04_UI_Catalog_AlertViews" time="0.0" classname="ios.scripts.Test_04_UI_Catalog_AlertViews">
          <skipped />
    </testcase>
  
  
</testsuite>
